apps/fz_http/lib/fz_http/gateways.ex

```diff
@@ -4,7 +4,31 @@ defmodule FzHttp.Gateways do
   """
 
   import Ecto.Query, warn: false
-  alias FzHttp.{Gateways.Gateway, Repo}
+  alias FzHttp.{Gateways.Gateway, Repo, Devices}
+
+  @default_action :deny
+  @default_name "default"
```

Comment:
I think this can cause some trouble, we want Gateway to identify itself with some unique generated ID and always pass it to the Panel in case it wants to be registered.

Otherwise, if we have multiple gateways registering themselves with the same token they will reuse the same configuration and there will be no way to distinguish between them. If that's something we want then we need to entities: GatewayGroup (with a user-friendly name and all Gateway configuration) and Gateway (IP, Version/User Agent, public key)

Reply:
(Will respond to this in the looong thread below)